src/openms/source/APPLICATIONS/ToolHandler.cpp

```diff
@@ -11,7 +11,7 @@
 #include <OpenMS/FORMAT/ToolDescriptionFile.h>
 #include <OpenMS/SYSTEM/File.h>
 
-#include <QtCore/QStringList>
+#include <QStringList>
 #include <QtCore/QDir>
```

Comment:
one in Core one not in Core?

Reply:
the prefixes are generally not required, but I left them in for clarity where possible.